frontend/common/errors.js

```diff
@@ -23,3 +23,14 @@ export class GIErrorUIRuntimeError extends Error {
     }
   }
 }
+
+export class GIErrorMissingSigningKeyError extends Error {
+  constructor (...params: any[]) {
+    super(...params)
+    // this.name = this.constructor.name
+    this.name = 'GIErrorMissingSigningKeyError' // string literal so minifier doesn't overwrite
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, this.constructor)
+    }
+  }
+}
```

Comment:
I think this is a good time to copy over that utility function from Chelonia to define these using 1-liners...

Reply:
Agreed.